src/App/_tests_/NotFound.test.js

```diff
@@ -5,6 +5,6 @@ import NotFound from '../Components/NotFound/NotFound';
 describe('NotFound', () => {
   const notFound = shallow(<NotFound />);
   it('renders the NotFound component', () => {
-    expect(notFound.find('Fragment').exists()).toBe(true);
+    expect(notFound.find('div').exists()).toBe(true);
```

Comment:
@MCFrank16 you're only testing that a `div` exists? what about the content of that div?